content/guides/daily-tasks/netservices.md

```diff
@@ -18,17 +18,19 @@ repeat new password:</pre></p>
 Haiku's default hostname is shredder. If shredder doesn't resolve, you can simply launch the
 <span class='app'>Network</span> preference applet to find your system's network address.
 You can change the hostname by editing /system/settings/network/hostname.</div>
+<br>
 
-<h3>ftpd - an FTP server daemon</h3>
+<h3>ftpd - FTP server daemon</h3>
 Haiku can easily be made to listen for incoming FTP connections to serve its files.
 
-<h4>Start ftpd at boot</h4>
-<p>Haiku will automatically start its FTP server daemon at boot if you remove the comment hashes (#) from the service ftp section of /boot/system/settings/network/services.
+<p>To start the FTP server select the Deskbar (the blue feather icon), Preferences, Network.  Select FTP Server and enable.</p>
```

Comment:
It's actually a blue leaf. :)
IMO we can assume the users know there way around Haiku's most basic usage, i.e. the Deskbar and how to launch apps and preferences with it.

```suggestion
<p>To start the FTP server open the Network preferences, select **`FTP server`**  from its list of services and click **`Enable`**.</p>
```

The same goes for the telnet server below.